test/ping-pong/proposer/src/routes/proposer.mjs

```diff
@@ -0,0 +1,32 @@
+/**
+Routes for managing a proposer.
+Some transactions are so simple that, we don't split out a separate service
+module but handle the entire request here.
+*/
+import express from 'express';
+import axios from 'axios';
+import logger from '../../../../../common-files/utils/logger.mjs';
+
+let optimistUrl = '';
+const router = express.Router();
+
+router.post('/offchain-transaction', async req => {
```

Comment:
Similar to how proposer interacts with optimist today via SDK, this will also be a function in SDK for the proposer to call the optimist. Since the SDK has the optimist url from nf3 class, it will be able to call this easily.